aggregates/governance-aggr/src/main/resources/db/store/h2/V0_1400_1__init.sql

```diff
@@ -56,3 +56,17 @@ create table gov_action_proposal_status
     update_datetime    timestamp,
     primary key (gov_action_tx_hash, gov_action_index, epoch)
 );
+
+drop table if exists drep_expiry;
```

Comment:
Can you add jooq entries for this new table in graalvm native image's resource-config.json file in this module? You can refer to other entries.. I think we need to add new generated pojo and table class.